middle_end/closure/closure.ml

```diff
@@ -900,6 +900,9 @@ let rec close ({ backend; fenv; cenv ; mutable_vars } as env) lam =
         | Const_pointer n -> Uconst_ptr n
         | Const_block (tag, fields) ->
             str (Uconst_block (tag, List.map transl fields))
+        | Const_float_block sl ->
+            (* CR mshinwell: Add [Uconst_float_block]? *)
```

Comment:
I don't think there's any real need to do this. I don't think Clambda needs to distinguish the two, and if one day it needs to it's easily changed then.

Reply:
This was a stale CR, removed.